### PR TITLE
Docs: Simple walkthrough for adding the Typesense Laravel Scout Driver to existing projects

### PR DESCRIPTION
**Pull request by tharropoulos**
## Change Summary
Adds a walkthrough for adding the Typesense Laravel Scout Driver to an existing project, from start to finish
## PR Checklist

- [x] I have read and signed the [Contributor License Agreement](https://forms.gle/PZyiY5N2GDQU8GsV9).

### CODE REVIEW

File: docs-site/content/guide/laravel-full-text-search.md [hunk 1]
```diff
@@ -0,0 +1,567 @@
...
+
+</Tabs>
+
+This command will create a new Eloquent model named `Game`, along with a migration file and a resource controller. Let's now modify the model file `app/Models/Game.php` to include the necessary columns for the Steam Games dataset:
+
+<Tabs :tabs="['PHP']">
+
+<template v-slot:PHP>
+
+```php{18,25-36,43-53}
```
Comment (jasonbosco) [inline docs-site/content/guide/laravel-full-text-search.md:236]: These highlights seem to be off by a few lines. Could you run the site locally and verify?

File: docs-site/content/guide/laravel-full-text-search.md [hunk 2]
```diff
@@ -0,0 +1,567 @@
...
+
+## Step 5: Index the Data Using Laravel Scout
+
+To index the data using Laravel Scout, let's add the `Searchable` trait to the `Game` model, and create a `toSearchableArray` method that returns the indexable data array for the model. This is required only for Models that have a different schema in the database than in Typesense, e.g. Models that include Dates. You can modify the `app/Models/Game.php` file as follows:
+
+<Tabs :tabs="['PHP']">
+
+<template v-slot:PHP>
+
+```php{7,44-55}
```
Comment (jasonbosco) [inline docs-site/content/guide/laravel-full-text-search.md:369]: These line highlights seem to highlight a few extra lines. Could you verify?

File: docs-site/content/guide/laravel-full-text-search.md [hunk 3]
```diff
@@ -0,0 +1,567 @@
...
+            ],
+        ],
+    ],
+```
+
+</template>
+
+</Tabs>
+
+After which, you can import the data into Typesense by running the following command:
```
Comment (jasonbosco) [inline docs-site/content/guide/laravel-full-text-search.md:526]: Could you add a section title here saying this is for Backfilling existing data. 

And just before that section, we want to say that once the Laravel Scout Typesense driver is setup, all changes to the models will be synced to Typesense automatically. 

And may be provide a code sample that shows modifying a DB record, and showing that it is reflected in Typesense by doing a search.

File: docs-site/content/guide/laravel-full-text-search.md [hunk 4]
```diff
@@ -0,0 +1,567 @@
...
+
+Game::search('persona')->get()->toArray();
+```
+
+</template>
+
+</Tabs>
+
+
+You can then proceed as you prefer. You can create a controller to handle the search requests, or use the [Typesense InstantSearch Adapter](https://github.com/typesense/typesense-instantsearch-adapter) to use Instantsearch.js on your frontend. If you prefer using a Javascript framework, you can use [Inertia.js](https://inertiajs.com/) to create a Vue.js, Svelte or React.js frontend. This [demo](https://github.com/typesense/showcase-laravel-steam-games-search/tree/master) uses a React Typescript frontend with the Typesense InstantSearch Adapter.
```
Comment (jasonbosco) [inline docs-site/content/guide/laravel-full-text-search.md:567]: Let's move the demos to under a new heading called "Demos".

**Review by jasonbosco — CHANGES_REQUESTED** [review]

File: docs-site/content/guide/laravel-full-text-search.md (hunk 1 above)
Comment (tharropoulos) [reply docs-site/content/guide/laravel-full-text-search.md:236]: This is how it looks on my end:
image 

Should I shift it up or down?

File: docs-site/content/guide/laravel-full-text-search.md (hunk 2 above)
Comment (tharropoulos) [reply docs-site/content/guide/laravel-full-text-search.md:369]: On this case, it doesn't seem to highlight the line itself, rather strange.
image

File: docs-site/content/guide/laravel-full-text-search.md (hunk 4 above)
Comment (tharropoulos) [reply docs-site/content/guide/laravel-full-text-search.md:570]: Could you please elaborate on this a bit more? I should add a new Heading called demos and include just the part that references the Demo, or move the whole paragraph over there?

File: docs-site/content/guide/laravel-full-text-search.md (hunk 3 above)
Comment (tharropoulos) [reply docs-site/content/guide/laravel-full-text-search.md]: The first issue is resolved in 72ecaa6185509485296c72359d080d49d8577b8c

About mentioning auto-syncing, it's being added in 8f267752fb9292da874edfac8da3127a72c17cbe

And finally, I added a small part showcasing real-time syncing through inserting a new record in the database in 2ae61d1c462d2a772770c67363546228b8c77c2b

File: docs-site/content/guide/laravel-full-text-search.md (hunk 1 above)
Comment (jasonbosco) [reply docs-site/content/guide/laravel-full-text-search.md:235]: Hmm, it looks fine here. It extended a couple of lines further down with each block...

File: docs-site/content/guide/laravel-full-text-search.md (hunk 4 above)
Comment (jasonbosco) [reply docs-site/content/guide/laravel-full-text-search.md:636]: Yeah, a new Heading called "Demo Apps" and include just the part that references the demo to it. This way it stands out in the sidenav bar

Something like this: https://typesense.org/docs/26.0/api/vector-search.html#ui-examples

**Comment by jasonbosco** [comment]
@tharropoulos Two more items I just remembered. 

1) Could you add a section about enabling queuing, since it's best to do that when using Typesense as the driver: https://laravel.com/docs/11.x/scout#queueing

2) Could you also mention dynamic search parameters: https://laravel.com/docs/11.x/scout#typesense-dynamic-search-parameters

File: docs-site/content/guide/laravel-full-text-search.md (hunk 4 above)
Comment (tharropoulos) [reply docs-site/content/guide/laravel-full-text-search.md]: > Yeah, a new Heading called "Demo Apps" and include just the part that references the demo to it. This way it stands out in the sidenav bar
> 
> Something like this: https://typesense.org/docs/26.0/api/vector-search.html#ui-examples

Addressed in 2ec91527da8972b0fd26cc48488785b60d19e1da

**Review by jasonbosco — APPROVED** [review]

**Pull request merged**